Makefile

```diff
@@ -42,6 +42,7 @@ rebuild:
 setup:
 	git submodule update --init --recursive
 	opam update -y
+	opam install -y --deps-only ./semgrep-core
 	opam install -y --deps-only ./pfff
 	cd ocaml-tree-sitter && ./configure && ./scripts/install-tree-sitter-lib
 	opam install -y --deps-only ./ocaml-tree-sitter
```

Comment:
I don't expect this to work because pfff and ocaml-tree-sitter are dependencies of semgrep-core. Maybe moving the line `opam install -y --deps-only ./semgrep-core` to last position works?

Reply:
I see it passes the tests. I don't know how this works. Is it due to the opam files found in `semgrep-core/`? In this case, it should be fine to remove `opam install -y --deps-only ./pfff` and `opam install -y --deps-only ./ocaml-tree-sitter`.